R/dgp-lib-errors.R

```diff
@@ -10,8 +10,10 @@
 #' 
 #' @return A vector of simulated errors with length \code{n}.
 #' 
-#' @details The function \code{err} must accept an argument named \code{n} 
-#'   (i.e., the sample size).
+#' @details The arguments \code{n} and \code{X} (if provided) are automatically
+#'   passed to the function \code{err} under arguments of the same name.
+#'   Note however that they may be unused arguments if \code{err} does not take 
+#'   in arguments named \code{n} and/or \code{X} as input.
```

Comment:
Since we're using `R.utils::doCall()`, I think this function still works and `n` and `X` both get passed through even if the `err()` function doesn't have `n`, `X`, or `...` as arguments. Do you think this is confusing or that we need to clarify this further in the docs?

Reply:
Ah, nice; wasn't familiar with `R.utils::doCall()`. Probably not necessary in that case.